components/tag/chip/src/index.js

```diff
@@ -6,8 +6,9 @@ const Tag = ({Link, children, url, ...rest} = {}) => url
   ? <Link href={url} {...rest}>{children}</Link>
   : <span {...rest}>{children}</span>
 
-const tagChipClassName = ({isClickable}) => cx('sui-TagChip', {
-  'sui-TagChip-link': isClickable
+const tagChipClassName = ({isClickable, className}) => cx('sui-TagChip', {
+  'sui-TagChip-link': isClickable,
+  [className]: className
```

Comment:
I think it's not needed. In any case, you can define a default prop as empty string.